File: 00-java_to_csharp/Manager.cs
using System;

namespace vererb1_java {
    public class Manager : Employee
    {
        public decimal WeeklySalary
        {
            set;
            get;
        }

        // Konstruktoren
        public Manager(String n) : base(n)
        {
            WeeklySalary = 0;
        }

        public Manager()
        {
            WeeklySalary = 0;
        }

        public override decimal ComputePay()
        {
            return WeeklySalary;
        }

    }
}

File: 00-java_to_csharp/Manager.java
package Angtst;

public class Manager extends Employee
{
	// Attribute um Angestelltendaten zu speichern
  private float weeklysalary;

           		// Konstruktoren
   public Manager(String n)
  {
  	super(n); // gib n an der Parentklassenkonstruktor weiter
        //System.out.println ("bin im Manager Param " + n);
    weeklysalary = 0;
  }

   public Manager()
  {
    weeklysalary = 0;
        //System.out.println ("bin im Managerkonstruktor " );
  }

   public void setweeklysalary (float w)
  {
        weeklysalary = w;
  }


  public float computePay()
  {
    return weeklysalary;
  }

}


